frontend/src/scenes/notebooks/Notebook/notebookActivityDescriber.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+    ActivityChange,
+    ActivityLogItem,
+    ActivityScope,
+    ChangeMapping,
+    Description,
+    HumanizedChange,
+} from 'lib/components/ActivityLog/humanizeActivity'
+import { SentenceList } from 'lib/components/ActivityLog/SentenceList'
+import { Link } from 'lib/lemon-ui/Link'
+import { urls } from 'scenes/urls'
+
+const notebookActionsMapping: Record<
+    string,
+    (change?: ActivityChange, logItem?: ActivityLogItem) => ChangeMapping | null
+> = {
+    version: () => null, // version will always change, but we don't show it to users
+    content: () => {
+        return {
+            description: [<>changed content</>],
+        }
+    },
+}
+
+function nameAndLink(logItem?: ActivityLogItem): JSX.Element {
+    return logItem?.detail?.short_id ? (
+        <Link to={urls.notebook(logItem.detail.short_id)}>{logItem?.detail.name || 'unknown'}</Link>
+    ) : logItem?.detail.name ? (
+        <>{logItem?.detail.name}</>
+    ) : (
+        <>unknown</>
+    )
+}
+
+export function notebookActivityDescriber(logItem: ActivityLogItem): HumanizedChange {
+    if (logItem.scope !== ActivityScope.NOTEBOOK) {
+        console.error('notebook describer received a non-Notebook activity')
```

Comment:
Should we capture this error anywhere?

Reply:
It's slightly laziness... it's there to help the future traveller adding a new kind of describer. _Really_ the problem should be refactored away